src/js/containers/covid19/recipient/RecipientContainer.jsx

```diff
@@ -35,7 +35,7 @@ const RecipientContainer = () => {
                     </p>
                     <p>
                         Please note that recipient information for the Small Business Administration (SBA) Paycheck Protection Program (PPP) is not available on USAspending but is available on SBA.gov.
-                        <a href="data/data-limitations.pdf" target="_blank" rel="noopener noreferrer"> Please see the &#34;Known Data Limitations&#34; PDF in the banner above for more information.</a>
+                        <a href="data/data-limitations.pdf" target="_blank" rel="noopener noreferrer"> Please see the &#34;Known Data Limitations&#34; PDF for more information.</a>
```

Comment:
Could we move the space between this and the previous sentence outside the link?